packages/calling/src/CallingClient/line/index.ts

```diff
@@ -204,4 +226,74 @@ export default class Line extends Eventing<LineEventTypes> implements ILine {
    */
   public getDeviceId = (): MobiusDeviceId | undefined =>
     this.registration.getDeviceInfo().device?.deviceId;
+
+  /**
+   * @param dest -.
+   */
+  public makeCall = (dest: CallDetails): ICall | undefined => {
+    let call;
+
+    // this is a temporary logic to get registration obj
```

Comment:
We don't need this comment here, please remove it

Reply:
Removed